types/angular/index.d.ts

```diff
@@ -1391,8 +1391,8 @@ declare namespace angular {
     interface IControllerService {
         // Although the documentation doesn't state this, locals are optional
         <T>(controllerConstructor: new (...args: any[]) => T, locals?: any, later?: boolean, ident?: string): T;
-        <T>(controllerConstructor: Function, locals?: IControllerLocals, later?: boolean, ident?: string): T;
-        <T>(controllerConstructor: Function, locals?: any, later?: boolean, ident?: string): T;
+        <T>(controllerConstructor: (...args: any[]) => T, locals?: IControllerLocals, later?: boolean, ident?: string): T;
+        <T>(controllerConstructor: (...args: any[]) => T, locals?: any, later?: boolean, ident?: string): T;
         <T>(controllerName: string, locals?: any, later?: boolean, ident?: string): T;
```

Comment:
While we're on it, please remove the undocumented parameters `later` and `ident` from all the signatures.